src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -212,8 +232,15 @@ export const NavBar: React.FC = track(
                 toggleMobileNav(showMenu)
               }}
             >
-              <MobileToggleIcon open={showMobileMenu} />
-              {showNotificationCount && <InboxNotificationCount />}
+              <Box
+                px={1}
+                display="flex"
+                alignItems="center"
+                justifyContent="center"
+              >
+                <MobileToggleIcon open={showMobileMenu} />
+                {showNotificationCount && <InboxNotificationCount />}
+              </Box>
```

Comment:
Moved the padding onto the inner element so the tap area is sensible. Problem with `NavItem`, really.